cli/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   markdown: ^4.0.0
   mustache_template: ^2.0.0
   path: ^1.8.0
+  shelf_static: ^1.1.0
```

Comment:
Sort formatting

Reply:
I'm not quite sure what you mean?

Reply:
deps should be sorted alphabethically

Reply:
is it already sorted, or am I missing something?

Reply:
Don't worry. Its resolved.